metrics-otlp/README.md

```diff
@@ -0,0 +1,134 @@
+    kubectl label namespace $KYMA_NS istio-injection=enabled
+    ```
+
+1. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+    helm repo update
+    ```
+
+## Install the gateway
+
+1. Deploy the gateway
+
+   Deploy an otel-collector using the upstream helm chart with prepared [values](./metrics-gateway-values.yaml). The values file is defining a pipeline for receiving OTLP metrics, is enriching them with resource attributes fullfilling the kubernetes semantic conventions, and then exports them to a custom OTLP backend.
+   
+   As this instructions are not providing any backend, the following command sets the backend configuration with placeholders `myEndpoint` and `myToken` which needs to get adjusted to your needs. Be aware of, that a token should be provided using a Secret which gets mounted via the `extraEnvs` parameter. As this cannot be passed via the command itself in an easy way, please consider using a dedicated additional values.yaml file for that.
+
+   ```bash
+   helm upgrade metrics-gateway open-telemetry/opentelemetry-collector --version 0.47.0 --install --namespace $KYMA_NS \
+     -f https://raw.githubusercontent.com/kyma-project/examples/main/metrics-otlp/metrics-gateway-values.yaml \
+     --set config.exporters.otlp.endpoint="{myEndpoint}" \
+     --set config.exporters.otlp.headers.Authorization="Bearer {myToken}"
+   ```
+
+1. Verify the deployment
+   Check that the related Pod has been created in the Namespace and is in the `Running` state:
+   ```bash
+   kubectl -n $KYMA_NS rollout status deploy metrics-gateway
+   ```
+
+## Install the agent
+
+1. Deploy the agent
+
+   The gateway deployed in the previous step can be used to push metrics via OTLP. If you prefer to use the prometheus pull-approach, a dedicated instance is needed for scraping your workload. Furthermore, an agent is required running as a daemonset in order to retrieve node specific metrics.
+   
+   The following command will deploy an otel-collector using the upstream helm chart with prepared [values](./metrics-agent-values.yaml). The values file is defining a metrics pipeline for scraping workload by annotation and pushing them to the gateway. It defines also a second metrics pipeline determining node-specific metrics for your workload from the nodes kubelet and the nodes filesystem itself.
+
+   ```bash
+   helm upgrade metrics-agent open-telemetry/opentelemetry-collector --version 0.47.0 --install --namespace $KYMA_NS \
+     -f https://raw.githubusercontent.com/kyma-project/examples/main/metrics-otlp/metrics-agent-values.yaml \
+     --set config.exporters.otlp.endpoint=metrics-gateway.$KYMA_NS:4317
+   ```
+
+1. Verify the deployment
+   Check that the related Pod has been created in the Namespace and is in the `Running` state:
+   ```bash
+   kubectl -n $KYMA_NS rollout status daemonset metrics-agent
+   ```
+
+## Usage
```

Comment:
```suggestion
## Result
```

This is simply the result of the instructions, right?  How to use the setup comes afterwards.